### PR TITLE
Increase username limit to 64 characters

### PR DESCRIPTION
**Pull request by lilcheti**

### CODE REVIEW

File: app/db/models.py
Comment (M03ED) [inline app/db/models.py:1]: it must be 64 not 66

**Review by M03ED — CHANGES_REQUESTED** [review]
database changes need migration file to sync database with code and you need to make and upload it with code 
also you did this change on master branch , changes and new features first come on dev branch then come to master branch

**Comment by kastov** [comment]
Does xray supports that lenth?

File: app/db/models.py
Comment (lilcheti) [reply app/db/models.py:1]: It was 34 how is that?

**Comment by kastov** [comment]
> > Does Xray-core supports that lenght?
> 
> A valid mail's username length is 64 characters 64 (Username) + 1 (@) + 255 (Domain Name) = 320 [i.e. example@localhost ]
> 
> How can Xray-core not accept that?

Not sure about 320 in total. Many sources say that the maximum length is around 254-256 characters.

**Comment by SaintShit** [comment]
this isn't enough, you need to provide a migration script too

**Pull request closed without merging**